rdkcentral/Lightning-UI-Components
# fix(Announcer): Ignore empty speech strings

## Description

Refactored the code to explicitly check whether the value of ProgressBar.announce and Slider.announce is undefined or null.
This change allows setting the value of ProgressBar.announce to an empty string and still getting an empty string instead of the actual value when the announce getter is called.

## References

LUI-792

## Testing

1. Navigate to the ScrollWrapper component.
2. Turn on the announcer.
3. The announcer should start announcing the text.
4. Slide the slider down.
5. The announcer should not announce the slider value, but continue announcing the text.

Also make sure this changes didn't effect ProgressBar and Slider announcer.
